pyfar/dsp/fft.py

```diff
@@ -5,7 +5,7 @@
 import multiprocessing
 
 import numpy as np
-from scipy import fft
+from scipy import fft as _fft
```

Comment:
Whats the reason for importing it as `_fft` instead of `fft`?

Reply:
* we need a function fft for computing double-sided Fourier spectra, this conflicts with the name of scipy.fft as fft.

Reply:
```suggestion
from scipy import fft as sfft
```